.pre-commit-config.yaml

```diff
@@ -32,3 +32,16 @@ repos:
         args:
           - --config-file
           - mypy.ini
+
+  - repo: https://github.com/python-jsonschema/check-jsonschema
+    rev: 0.15.0
+    hooks:
+      - id: check-metaschema
+        name: "Check JSON schemas validity"
+        files: ^schemas/.*\.yaml
```

Comment:
The directory has been moved under `tmt/`

Reply:
oh indeed, thx!